werkzeug/serving.py

```diff
@@ -263,10 +263,10 @@ def version_string(self):
         return BaseHTTPRequestHandler.version_string(self).strip()
 
     def address_string(self):
-        if getattr(self, 'environ', None):
-            return self.environ['REMOTE_ADDR']
```

Comment:
Disregarding `REMOTE_ADDR` makes `werkzeug.contrib.fixers.ProxyFix` useless (at least when it comes to logging).
Couldn't we try `getattr(self, 'environ', None)` as we previously did and only fall back to `self.client_address[0]` when that fails?